shopfloor/services/checkout.py

```diff
@@ -1749,6 +1826,7 @@ def _schema_stock_picking_details(self):
             group_lines_by_location={"type": "boolean"},
             show_oneline_package_content={"type": "boolean"},
             need_confirm_pack_all={"type": "string"},
+            need_confirm_lot={"type": "integer"},
```

Comment:
The v14 PR was adding `nullable: True` here, removed on purpose?

EDIT: ok you switched from None to False everywhere

Reply:
I do not remember why it was put as `integer+nullable` instead of boolean... like if there was a trap there